pkg/apis/core/types.go

```diff
@@ -6493,3 +6507,19 @@ const (
 	// the destination set to the node's IP and port or the pod's IP and port.
 	LoadBalancerIPModeProxy LoadBalancerIPMode = "Proxy"
 )
+
+// ImageVolumeSource represents a image volume resource.
+type ImageVolumeSource struct {
+	// Required: Image or artifact reference to be used.
+	// Behaves in the same way as pod.spec.containers[*].image.
+	// More info: https://kubernetes.io/docs/concepts/containers/images
```

Comment:
should we add `// +required` here?

Reply:
It does not change the public API if we add that comment, neither to this file nor `staging/src/k8s.io/api/core/v1/types.go`. So I'd prefer to keep the `Required:` in the comment.